src/main/java/ch/njol/skript/effects/EffFeed.java

```diff
@@ -25,52 +25,55 @@
 import ch.njol.skript.doc.Since;
 import ch.njol.skript.lang.Effect;
 import ch.njol.skript.lang.Expression;
-import ch.njol.skript.lang.SkriptParser;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
 import ch.njol.util.Kleenean;
 import org.bukkit.entity.Player;
 import org.bukkit.event.Event;
 import org.eclipse.jdt.annotation.Nullable;
 
 @Name("Feed")
 @Description("Feeds the specified players.")
-@Examples({"feed all players", "feed the player by 5 beefs"})
+@Examples({
+	"feed all players",
+	"feed the player by 5 beefs"
+})
 @Since("2.2-dev34")
 public class EffFeed extends Effect {
 
     static {
-        Skript.registerEffect(EffFeed.class, "feed [the] %players% [by %-number% [beef[s]]]");
+        Skript.registerEffect(EffFeed.class, "feed [the] %players% [by %-number% [beef[s]|hunger]]");
     }
 
-    @SuppressWarnings("null")
+	@SuppressWarnings("NotNullFieldNotInitialized")
     private Expression<Player> players;
     @Nullable
-    private Expression<Number> beefs;
+    private Expression<Number> hunger;
 
     @Override
-    public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, SkriptParser.ParseResult parseResult) {
+	@SuppressWarnings("unchecked")
+    public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, ParseResult parseResult) {
         players = (Expression<Player>) exprs[0];
-        beefs = (Expression<Number>) exprs[1];
+        hunger = (Expression<Number>) exprs[1];
         return true;
     }
 
     @Override
-    protected void execute(Event e) {
+    protected void execute(Event event) {
         int level = 20;
-
-        if (beefs != null) {
-            Number n = beefs.getSingle(e);
-            if (n == null)
+        if (hunger != null) {
+            Number number = hunger.getSingle(event);
+            if (number == null)
                 return;
-            level = n.intValue();
+            level = number.intValue();
         }
-        for (Player player : players.getArray(e)) {
-            player.setFoodLevel(player.getFoodLevel() + level);
+        for (Player player : players.getArray(event)) {
+            player.setFoodLevel(Math.min(20, player.getFoodLevel() + level));
```

Comment:
I have a thought of following the rule of how Skript sets health, which is the ratio of 1:2 to Minecraft's — 0.5 hunger in Skript : 1 hunger in Minecraft.
Looking for thoughts on this, although I believe this will be a breaking change.

Reply:
This is already the case for ExprFoodLevel, this should be the same.

Reply:
See https://github.com/SkriptLang/Skript/blob/302f8f9ac7cac1d1a1a68ce85cb75fad4fafd4a1/src/main/java/ch/njol/skript/bukkitutil/HealthUtils.java#L48